### PR TITLE
Fix postgres integration tests

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -799,8 +818,9 @@ steps:
     apps/notifications
   - ./occ app:enable notifications
   - cd apps/$APP_NAME
+  - composer install --no-dev
   - cd tests/integration/
-  - bash run.sh features/chat
+  - bash run.sh features/chat/message-expiration.feature
```
Comment (nickvergessen) [inline .drone.yml:823]: Needs reverting after tests passed

File: lib/Manager.php [hunk 2]
```diff
@@ -222,7 +222,7 @@ public function createCommentObject(array $row): ?IComment {
 			'creation_timestamp' => $row['comment_creation_timestamp'],
 			'latest_child_timestamp' => $row['comment_latest_child_timestamp'],
 			'reactions' => $row['comment_reactions'],
-			'expire_date' => $row['expire_date'],
+			'expire_date' => $row['comment_expire_date'],
```
Comment (nickvergessen) [inline lib/Manager.php:225]:
```json
{"Exception":"Error","Message":"Undefined array key \"expire_date\" at \/drone\/server\/apps\/spreed\/lib\/Manager.php#225","Code":0,"Trace":[{"file":"\/drone\/server\/apps\/spreed\/lib\/Manager.php","line":225,"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"file":"\/drone\/server\/apps\/spreed\/lib\/Manager.php","line":150,"function":"createCommentObject","class":"OCA\\Talk\\Manager","type":"->"},{"file":"\/drone\/server\/apps\/spreed\/lib\/Manager.php","line":343,"function":"createRoomObject","class":"OCA\\Talk\\Manager","type":"->"},{"file":"\/drone\/server\/apps\/spreed\/lib\/Manager.php","line":300,"function":"getRoomsForActor","class":"OCA\\Talk\\Manager","type":"->"},{"file":"\/drone\/server\/apps\/spreed\/lib\/Controller\/RoomController.php","line":194,"function":"getRoomsForUser","class":"OCA\\Talk\\Manager","type":"->"},{"file":"\/drone\/server\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":225,"function":"getRooms","class":"OCA\\Talk\\Controller\\RoomController","type":"->"},{"file":"\/drone\/server\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":133,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"\/drone\/server\/lib\/private\/AppFramework\/App.php","line":172,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"\/drone\/server\/lib\/private\/Route\/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"\/drone\/server\/ocs\/v1.php","line":62,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"\/drone\/server\/ocs\/v2.php","line":23,"args":["\/drone\/server\/ocs\/v1.php"],"function":"require_once"}],"File":"\/drone\/server\/lib\/private\/Log\/ErrorHandler.php","Line":92,"CustomMessage":"--"}
```

File: tests/integration/spreedcheats/lib/Controller/ApiController.php [hunk 3]
```diff
@@ -132,12 +132,16 @@ public function getMessageExpirationJob($token): DataResponse {
...
-	private function getRoomIdByToken(string $token): ?string {
+	private function getRoomIdByToken(string $token): ?int {
 		$query = $this->db->getQueryBuilder();
 		$query->select('id')
 			->from('talk_rooms')
 			->where($query->expr()->eq('token', $query->createNamedParameter($token)));
+
 		$result = $query->executeQuery();
-		return $result->fetchOne();
+		$roomId = (int) $result->fetchOne();
```
Comment (nickvergessen) [inline tests/integration/spreedcheats/lib/Controller/ApiController.php:142]:
```json
{"Exception":"Exception","Message":"OCA\\SpreedCheats\\Controller\\ApiController::getRoomIdByToken(): Return value must be of type ?string, int returned in file '\/drone\/server\/apps\/spreedcheats\/lib\/Controller\/ApiController.php' line 141","Code":0,"Trace":[{"file":"\/drone\/server\/lib\/private\/AppFramework\/App.php","line":172,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"\/drone\/server\/lib\/private\/Route\/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"\/drone\/server\/ocs\/v1.php","line":62,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"\/drone\/server\/ocs\/v2.php","line":23,"args":["\/drone\/server\/ocs\/v1.php"],"function":"require_once"}],"File":"\/drone\/server\/lib\/private\/AppFramework\/Http\/Dispatcher.php","Line":165,"Previous":{"Exception":"TypeError","Message":"OCA\\SpreedCheats\\Controller\\ApiController::getRoomIdByToken(): Return value must be of type ?string, int returned","Code":0,"Trace":[{"file":"\/drone\/server\/apps\/spreedcheats\/lib\/Controller\/ApiController.php","line":114,"function":"getRoomIdByToken","class":"OCA\\SpreedCheats\\Controller\\ApiController","type":"->"},{"file":"\/drone\/server\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":225,"function":"getMessageExpirationJob","class":"OCA\\SpreedCheats\\Controller\\ApiController","type":"->"},{"file":"\/drone\/server\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":133,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"\/drone\/server\/lib\/private\/AppFramework\/App.php","line":172,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"\/drone\/server\/lib\/private\/Route\/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"\/drone\/server\/ocs\/v1.php","line":62,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"\/drone\/server\/ocs\/v2.php","line":23,"args":["\/drone\/server\/ocs\/v1.php"],"function":"require_once"}],"File":"\/drone\/server\/apps\/spreedcheats\/lib\/Controller\/ApiController.php","Line":141},"CustomMessage":"--"}
```

**Comment by nickvergessen** [comment]
Tests are finally green, please review so I can revert the postgres drone file change and merge the PR tomorrow morning after CI is done.

cc @danxuliu @vitormattos

**Review by danxuliu — APPROVED** [review]
Changes look good :+1: 

But please drop the _Debug_ commit with an interactive rebase and force push rather than just revert it :-P

**Comment by nickvergessen** [comment]
> But please drop the Debug commit with an interactive rebase and force push rather than just revert it :-P

That is what reverting is to me unless it was merged into master/main :-P

File: .drone.yml (hunk 1 above)
Comment (nickvergessen) [reply .drone.yml]: Dropped the commit

**Pull request merged**